pkg/auth/authz.go

```diff
@@ -55,3 +58,17 @@ func NewAuthzHandler(cfg *Config, ocmCLient *ocm.Client, log logrus.FieldLogger,
 	}
 	return &NoneHandler{}
 }
+
+func ValidateAccessToMultiarch(ctx context.Context, authzHandler Authorizer) error {
+	var err error
+	var multiarchAllowed bool
+
+	multiarchAllowed, err = authzHandler.HasOrgBasedCapability(ctx, ocm.MultiarchCapabilityName)
+	if err != nil {
+		return common.NewApiError(http.StatusInternalServerError, fmt.Errorf("error getting user %s capability, error: %w", ocm.MultiarchCapabilityName, err))
+	}
+	if !multiarchAllowed {
+		return common.NewApiError(http.StatusBadRequest, errors.Errorf("%s", "multiarch clusters are not available"))
```

Comment:
404 will be more appropriate here

Reply:
How about making it part of hasAccessTo with an object that represent a multiarch. and the application level can then invoke it pretty much as it invokes ValidateAccessToMultiarch.

Reply:
I can change it, but this is just a copy and paste.

Reply:
I think I'd rather just remove the function entirely. It's just some error handling wrapping a single call.

One caller is matching based on the error string so it would be better for it to be able to define how it handles errors. Then the other caller could implement this function or not.